Update `p2p-private-key-file` command line option description (#546)
Consensys/doc.teku: docs/reference/cli/index.md

| `gnosis` | Consensus layer | Production | Network for the [Gnosis chain](https://www.gnosis.io/) |
| `holesky` | Consensus layer | Test | Multi-client testnet |
| `sepolia` | Consensus layer | Test | Multi-client testnet |

> | `chiado` | Consensus layer | Test | Gnosis [testnet](https://docs.gnosischain.com/about/networks/chiado/) |

Reviewer: no link for this

File containing the [node's private key](../../concepts/p2p-private-key.md).

> If there is not a file present at the path provided, one will be created with a new p2p private key generated and stored inside.

Reviewer: ```suggestion
If a file doesn't exist at the specified path, Teku creates a new file and p2p private key to store inside.
```

:::important

> Ensure you provide a file path and not a directory as no file will be read or created.

Reviewer: ```suggestion
Make sure you specify a file path and not a directory. Teku doesn't read or create files in directories.
```

Author: I mean it does create files in directories, but you need to specify the path with the file name, not just the directory it is in.

Reviewer: oh! Thank you for the clarification @courtneyeh
